xml/installation.xml

```diff
@@ -29,143 +26,31 @@
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Includes &hpca;-enabled libraries
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Disables firewall and Kdump services
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Installs a client for the &slurm; workload manager
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Does not create a separate <literal>/home</literal> partition
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Mounts a large scratch partition to <filename>/var/tmp</filename>
-        </para>
-       </listitem>
-      </itemizedlist>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>&hpca; development node</term>
-     <listitem>
-      <para>
-       This role includes the following features:
-      </para>
-      <itemizedlist>
-       <listitem>
-        <para>
-         Includes &hpca;-enabled libraries
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Adds compilers and development toolchains
-        </para>
-       </listitem>
-      </itemizedlist>
-     </listitem>
-    </varlistentry>
-   </variablelist>
+   <itemizedlist>
+    <listitem>
+     <para>
+      <literal>Development Tools Module</literal>
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <literal>HPC Module</literal>
+     </para>
+    </listitem>
+   </itemizedlist>
    <para>
-    The scratch partition <literal>/var/tmp/</literal> is only created if
-    there is sufficient space available on the installation medium (minimum
-    32&nbsp;GB).
-   </para>
-   <para>
-    The environment module <literal>Lmod</literal> is installed for all
-    roles. It is required at build time and runtime of the system. For more
-    information, see <xref linkend="sec-compute-lmod"/>.
-   </para>
-   <para>
-    All libraries specifically built for &hpca; are installed under
-    <literal>/usr/lib/hpc</literal>. They are not part of the standard search
-    path, so the <literal>Lmod</literal> environment module system is
-    required.
-   </para>
-   <para>
-    <literal>&munge;</literal> authentication is installed for all roles. &munge;
-    keys are generated and must be copied to all nodes in the
-    cluster. For more information, see <xref linkend="sec-remote-munge"/>.
-   </para>
-   <para>
-    The system roles are only available for new installations of the &hpcm; for &sles;.
+    Use either the <literal>Text Mode</literal> or <literal>Minimal</literal> system role.
    </para>
   </sect1>
   <sect1 xml:id="sec2-upgrade">
    <title>Upgrading to &sles; &hpcm; &productnumber;</title>
    <para>
-    In version 15 SP6, &slehpc; (&slehpca;) was removed as a separate product, and is now available as the &hpcm; for &sles; (&slsa;).
+    In version 15 SP6, &slehpc; (&slehpca;) was removed as a separate product, and is
```

Comment:
It should read something like this:
Since the SLE HPC Product has been removed for SP6 you will be offered to migrate to SLES 15 SP6 hen upgrading from a SLE HPC. This migration will retain all the software modules you had enabled on SLE HPC as well as all software packages. After the migration the same software selection will be available as on SLE HPC.